android/CONTRIBUTING.md

```diff
@@ -1,52 +1,113 @@
-* 🔬 State what behavior you expected and what the actual behavior is on your
-  device
+We welcome all types of contributions, including, but not limited to:
+* [Filing issues](#filing-issues)
+* [Translating our apps](#translating-our-apps)
+* [User experience design](#user-experience-design)
+* [Writing documentation](#documentation)
+* [Testing the app](#testing-the-app)
+* [Writing code](#writing-code)
 
-## Writing code
+Have another idea for how you can contribute? Let us know!
 
-* Start by looking for issues marked with the [`good first issue`
-  label](https://github.com/mozilla-mobile/focus-android/labels/good%20first%20issue)
-* You can find more challenging issues marked with the [`help wanted`
-  label](https://github.com/mozilla-mobile/focus-android/labels/help%20wanted)
-* Join the #focus channel on [irc.mozilla.org](https://wiki.mozilla.org/IRC)
-  and get in contact with us. We're available Monday-Friday, during GMT and PST
-working hours.
-* Comment on the issue if you would like to work on it.
-* If you want to work on a new feature then always file an issue first so that
-  all teams (product, ux, engineering) can comment on it and so that it can be
-assigned to a milestone. **Pull requests for unsolicited features are unlikely
-to get merged.**
-* When you open a pull request, please also include a screenshot if there are
-  UI changes, so UX can also do a visual review.
-* Include a `Closes #<issue-number>` as part of your first commit message so
-  it's auto-linked to the issue.
+### Communication
+For each our projects, we communicate with the project's:
+- Issues tracker
+- Mailing list
+- [IRC](https://wiki.mozilla.org/IRC) channel. *We're available Monday-Friday
+during GMT and PST working hours.*
+
+**Be sure to join them** and don't be afraid to make postings of your
+own! A link to each project-specific communication channel can be found in
+the project's README.
+
+## Filing issues
+
+We usually track our project's issues on GitHub (example: [focus-android][fa issues]).
+To file a bug:
+* Find your project's issues: these will be linked from the project's README
+or you can click directly into the issues from the project's home page
+* Create a new issue!
 
-## Translating the app
+To ensure we can understand, reproduce, and effectively triage your issues,
+**please fill out the provided template** when filing an issue! When issues
+are filed in a consistent style and include the information we need to
+make decisions, we're more likely to take action on these issues than if
+we don't understand them.
+
+*Be sure to include any extra information you find valuable!* For example, posting
+[your user agent](https://duckduckgo.com/?q=my+user+agent) can help us debug problems.
+
+## Translating our apps
 
 * Get in touch with the [existing localization
   team](https://wiki.mozilla.org/L10n:Teams).
 * Localization happens on
   [Pontoon](https://pontoon.mozilla.org/projects/focus-for-android/).
```

Comment:
Nit: this links to focus's l10n repo - probably change this to be an example too, or just the projects/ url if that makes sense.